x-pack/winlogbeat/module/sysmon/config/winlogbeat-sysmon.js

```diff
@@ -331,10 +331,21 @@ var sysmon = (function () {
             evt.Delete("user");
             evt.Put("user.domain", userParts[0]);
             evt.Put("user.name", userParts[1]);
+            evt.Put("related.user", [userParts[1]]);
```

Comment:
safer to use evt.AppendTo, just in case in the future something sets this before you do since related.user is an array.